src/snowflake/snowpark/modin/plugin/compiler/snowflake_query_compiler.py

```diff
@@ -8435,6 +8435,30 @@ def apply(
                 )
             return self._apply_snowpark_python_function_to_columns(func, kwargs)
 
+        # TODO SNOW-1739034: remove 'no cover' when apply tests are enabled in CI
+        sf_func = NUMPY_UNIVERSAL_FUNCTION_TO_SNOWFLAKE_FUNCTION.get(
+            func
+        )  # pragma: no cover
+        if sf_func is not None:  # pragma: no cover
+            return self._apply_snowpark_python_function_to_columns(sf_func, kwargs)
+
+        if get_snowflake_agg_func(func, {}, axis) is not None:  # pragma: no cover
+            # np.std and np.var 'ddof' parameter defaults to 0 but
+            # df.std and df.var 'ddof' parameter defaults to 1.
+            # Set it here explicitly to 0 if not provided.
+            if func in (np.std, np.var) and "ddof" not in kwargs:
+                kwargs["ddof"] = 0
+            # np.median return NaN if any value is NaN while df.median skips NaN values.
+            # Set 'skipna' to false to match behavior.
```

Comment:
so for median, we will use np.median behavior?

Reply:
Yes, default behavior for np.median is skipna == False. We need to map this to df.median/ df.agg('median') that by default is skipna == True. So we need to change this here so we can reuse existing code which implements df.agg('median').

Reply:
ok, so func is the targeted function we need to mapped to